Project2/style.css

```
@@ -0,0 +1,25 @@
+.card {
+    max-width: 500px;
+    margin: 10px auto;
+    background-color: #ffffff;}
+figure {margin: 0 auto;}
+        max-width: 500px;
+.button1 {
+    background-color: #008CBA;
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+  }
+  body {margin: 10px
+    }
+    .thumbnail {
+        max-width: 500px; 
```

Rather than a specific number here, you can use this percentage:
```suggestion
        max-width: 100%; 
```
This has the same effect, but is also more flexible because you wouldn't have to update it if you changed the card size.